pkg/mounter/safe_mounter_host_process_windows.go

```diff
@@ -0,0 +1,268 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mounter
+
+import (
+	"context"
+	"fmt"
+	"os"
+	filepath "path/filepath"
+	"strings"
+
+	"k8s.io/klog/v2"
+	mount "k8s.io/mount-utils"
+
+	"sigs.k8s.io/azurefile-csi-driver/pkg/os/filesystem"
+	"sigs.k8s.io/azurefile-csi-driver/pkg/os/smb"
+)
+
+var _ CSIProxyMounter = &winNativeCallMounter{}
+
+type winNativeCallMounter struct{}
+
+func NewWinNativeCallMounter() *winNativeCallMounter {
+	return &winNativeCallMounter{}
+}
+
+func (mounter *winNativeCallMounter) SMBMount(source, target, fsType string, mountOptions, sensitiveMountOptions []string) error {
```

Comment:
rename as `winMounter`, that's already different from `csiProxyMounter`

Reply:
done